### PR TITLE
updating java option for gateway

### PR DESCRIPTION
**Pull request by jeanmi151**
make possible to customize JAVA_TOOL_OPTIONS and JAVA_OPTS var

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -116,6 +116,7 @@ georchestra:
     gateway:
       enabled: false
       docker_image: georchestra/gateway:latest
+      java_tool_options: "-Dspring.config.location=${georchestra.datadir}/gateway/application.yaml -Dspring.profiles.active=docker"
```
Comment (pmauduit) [inline values.yaml:119]: missing the `-Dgeorchestra.datadir=...` which disappeared from the deployment spec

**Review by pmauduit — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 1 above)
Comment (jeanmi151) [reply values.yaml]: good catch

File: values.yaml [hunk 2]
```diff
@@ -116,6 +116,7 @@ georchestra:
     gateway:
       enabled: false
       docker_image: georchestra/gateway:latest
+      java_tool_options: "-Dgeorchestra.datadir=/etc/georchestra -Dspring.config.location=${georchestra.datadir}/gateway/application.yaml -Dspring.profiles.active=docker"
```
[the remark this answers is not in the document]
Comment (jeanmi151) [reply values.yaml:119]: good catch

File: Chart.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.10.0
+version: 0.10.1
```
Comment (edevosc2c) [inline Chart.yaml:18]: I would prefer to bump the version number in another commit.

Especially this is a wrong version number bump according to https://semver.org/

**Review by edevosc2c — CHANGES_REQUESTED** [review]

**Comment by edevosc2c** [comment]
Why this needs to be customized? Isn't it always the same JAVA parameters used?

File: Chart.yaml (hunk 3 above)
Comment (jeanmi151) [reply Chart.yaml:18]: I will squash and merge any way for merge pr

Comment (jeanmi151) [reply Chart.yaml:18]: je dirai que c'est du patch de gateway donc c'est bon pou rla version

Comment (edevosc2c) [reply Chart.yaml:18]: no we had several new changes done by jean pommier last week, these are new features.
Please remove the version number bump in this PR I'll create another commit for that.

File: values.yaml (hunk 2 above)
Comment (edevosc2c) [inline values.yaml:119]: Could you put this parameter under a new section called "environment"?

Like so:
```
environment:
  JAVA_TOOL_OPTIONS: "-Dgeorchestra.datadir=/etc/georchestra -Dspring.config.location=${georchestra.datadir}/gateway/application.yaml -Dspring.profiles.active=docker"
```

This way it's categorized inside a specific section.

This makes me realized that we are missing the special `extra_environment: []` too.

**Review by edevosc2c — APPROVED** [review]

**Pull request merged**